Joe-submission-info command Update (#34694)
demisto/content: Packs/JoeSecurity/ReleaseNotes/1_1_21.md

#### Integrations

##### Joe Security v2

> - Updated the **joe-submission-info** command to requires the *submission_ids* argument to be provided.

Reviewer: ```suggestion
- The *submission_ids* argument is now required when using the command **joe-submission-info**.
```
